searchcore/src/vespa/searchcore/proton/server/documentdb.h

```diff
@@ -297,11 +297,12 @@ class DocumentDB : public DocumentDBConfigOwner,
     size_t getNumDocs() const;
 
     /**
-     * Returns the number of documents that are active for search in this database.
+     * Returns the number of documents that are active for search in this database,
+     * and the number of documents that will be active once ideal state is reached.
      *
-     * @return The active-document count.
+     * @return The active and target-active document count.
      */
-    size_t getNumActiveDocs() const;
+    std::pair<size_t, size_t> getNumActiveDocs() const;
```

Comment:
@baldersheim Consider adding a struct for this, to avoid confusion of what is _active_ and _target_active_.

Reply:
See #24027 for followup